website/pages/docs/configuration/consul.mdx

```diff
@@ -104,6 +104,12 @@ configuring Nomad to talk to Consul via DNS such as consul.service.consul
   Consul service name defined in the `server_service_name` option. This search
   only happens if the server does not have a leader.
 
+- `share_ssl` `(bool: true)` - Specifies whether the Nomad client should share
+  its Consul SSL configuration with Connect Native applications. Includes values
+  of `ca_file`, `cert_file`, `key_file`, `auth`, `ssl`, and `verify_ssl`. Does
+  not include the value for the ACL `token`. This option should be disabled in
+  an untrusted environment.
```

Comment:
I don't like "untrusted environment" as we don't define what that means and are moving toward trusting individual jobs less and less.

My understanding is that as long as Consul ACLs are enabled, sharing TLS certificates does not meaningfully reduce the overall security of Nomad+Consul as the native task will only have its SI token's permissions.

Perhaps stating that this option should be _disabled_ if Consul ACLs are _disabled_ might be less vague and more prescriptive?

Reply:
When `share_ssl` is enabled, Connect Native tasks are granted the same level of authenticity as the Nomad Client agent (i.e. sharing certificates). They are scoped down in authorization to the default Consul Service Identity [Virtual] Policy tokens - or whatever consul token is injected through `env` or `template`. 

Turning this feature off makes sense if the security of your cluster depends entirely on authentication (you have HTTPS enabled but not ACLs).

I'll try to clarify that better.